ament_lint_auto/cmake/ament_lint_auto_find_test_dependencies.cmake

```diff
@@ -17,15 +17,24 @@
 #
 # All found package names are appended to the
 # ``${PROJECT_NAME}_FOUND_TEST_DEPENDS`` variables.
+# :param EXCLUDE: List of linter packages that should be skipped
+# :type EXCLUDE: list of strings
```

Comment:
Based on the docs I would expect that the listed packages are also not being `find_package()`-ed.

Reply:
We would have to go back to the messy `set(<package>_FOUND TRUE)` solution that I first proposed (https://github.com/ament/ament_lint/pull/133#discussion_r268344474)... maybe it's better to just let users set the variable outside of `ament_lint_auto_find_test_dependencies`?

I'm just not sure where to document that this variable exists and how to use it.

Reply:
I keep forgetting that the test dependencies are not necessarily immediately the linter packages. :thinking:

Since the function `ament_lint_auto_find_test_dependencies` doesn't do anything beside setting a global variable I would lean towards not adding the argument to it.

> maybe it's better to just let users set the variable outside of `ament_lint_auto_find_test_dependencies`?
 > I'm just not sure where to document that this variable exists and how to use it.

Let's do that. Since there is no direct function being called there is no good docblock to put the docs. Maybe just put it into `ament_lint_auto_package_hook.cmake` where it is going to be used.